src/openrct2/drawing/Drawing.h

```diff
@@ -13,6 +13,7 @@
 #include "../common.h"
 #include "../interface/Colour.h"
 #include "../interface/ZoomLevel.hpp"
+#include "../world/Location.hpp"
```

Comment:
You don't need to include this header if you forward declare the screencoordsxy.

`struct ScreenCoordsXY;`